api/lib/application/cache/cache-controller.js

```diff
@@ -3,9 +3,9 @@ const _ = require('lodash');
 
 module.exports = {
 
-  refreshCacheEntries() {
-    return Promise.all(_.map(AirtableDatasources, (datasource) => datasource.refreshAirtableCacheRecords()))
-      .then(() => null);
+  refreshCacheEntries(request, h) {
+    _.forEach(AirtableDatasources, (datasource) => datasource.refreshAirtableCacheRecords());
+    return h.response().code(204);
```

Comment:
C'est peut-être enfin l'occasion d'utiliser [202](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/202) !

Reply:
Bien vu !
- "HTTP Status 202 indicates that request has been accepted for processing, but the processing has not been completed. This status code is useful when the actual operation is asynchronous in nature"
- "HTTP Status 204 (No Content) indicates that the server has successfully fulfilled the request and that there is no content to send in the response payload body"